alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportGetMonitorAction.kt

```diff
@@ -115,4 +137,41 @@ class TransportGetMonitorAction @Inject constructor(
             )
         }
     }
+
+    private fun getAssociatedWorkflows(id: String): List<AssociatedWorkflow> {
+        try {
+            val associatedWorkflows = mutableListOf<AssociatedWorkflow>()
+            val queryBuilder = QueryBuilders.nestedQuery(
+                TransportDeleteWorkflowAction.WORKFLOW_DELEGATE_PATH,
+                QueryBuilders.boolQuery().must(
+                    QueryBuilders.matchQuery(
+                        TransportDeleteWorkflowAction.WORKFLOW_MONITOR_PATH,
```

Comment:
Ideally the `WORKFLOW_DELEGATE_PATH` and `WORKFLOW_MONITOR_PATH` constants should be in a central constants file for workflows.

Reply:
moved to utils class